tests/contrib/patch.py

```diff
@@ -148,6 +149,48 @@ def patch(self, *args, **kwargs):
             from ddtrace import patch
             return patch(*args, **kwargs)
 
+        def _gen_test_attrs(self, ops):
+            """
+            A helper to return test names for tests given a list of different
+            operations.
+            :return:
+            """
+            from itertools import permutations
+            return [
+                'test_{}'.format('_'.join(c)) for c in permutations(ops, len(ops))
+            ]
+
+        def test_averify_test_coverage(self):
```

Comment:
hah! this was so that it would run first and save some debugging time 😆

Reply:
You could just run these assertions in `setUpClass` ?

Failing to run any tests because they are not all there is the same in by book as having 1 failing test for not having all of them defined.

Reply:
although, if someone overrides `setUpClass` without calling `super()`, then that sucks.